src/crawlee/base_storage_client/base_request_queue_client.py

```diff
@@ -185,17 +184,3 @@ async def batch_delete_requests(self, requests: list[Request]) -> BatchRequestsO
         Args:
             requests: The requests to delete from the queue.
         """
-
-    @abstractmethod
-    async def list_requests(
```

Comment:
Yes, it's not used in the JS version either. We'd like to make an interface for storage clients that reflects what Crawlee needs, instead of just blindly copying the Apify platform API.